cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -731,15 +731,15 @@ DefinitionByReferenceNode definitionByReferenceNodeFromArgument(Expr argument) {
 }
 
 /** Gets the `VariableNode` corresponding to the variable `v`. */
-VariableNode variableNode(Variable v) { result.getVariable() = v }
+deprecated VariableNode variableNode(Variable v) { result.getVariable() = v }
```

Comment:
This predicate, and the other unused `somethingNode` predicates, exist for consistency: if there's a public `SomethingNode` class, then there's a public `somethingNode` predicate.